met-web/src/App.tsx

```diff
@@ -94,10 +94,12 @@ const App = () => {
 
     const getTranslationFile = async () => {
         try {
-            const translationFile = await import(`./locales/${language}/${tenant.id}.json`);
+            const translationFile = await import(`./locales/${language}/${basename}.json`);
+            document.title = translationFile?.header?.title || document.title;
```

Comment:
ok so maybe we need to leverage React Helmet to do something like this, it's probably cleaner. Can you check the library